index.bs

```diff
@@ -575,79 +575,225 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 		<div id="navigator-idl" dfn-for="Navigator">
 
 		<div class="algorithm" data-algorithm="navigator-clipboard">
-		<h4 id="h-navigator-clipboard"><dfn>clipboard</dfn></h4>
-		The [=clipboard=] attribute must return the {{Navigator}}'s [=clipboard=] object.
+		<h4 id="h-navigator-clipboard">clipboard</h4>
+		It is used to execute read/write operations from/to the [=system clipboard=].
+		The {{Navigator/clipboard}} getter steps are to return [=this=]'s [=clipboard=] object.
 		</div>
 
 		</div><!-- dfn-for Navigator -->
 
-	<h3 id="clipboard-interface">Clipboard Interface</h3>
-
+	<h3 id="clipboard-item-interface">ClipboardItem Interface</h3>
 		<pre class="idl" data-highlight="webidl">
-		typedef sequence&lt;ClipboardItem> ClipboardItems;
+			typedef (DOMString or Blob) ClipboardItemDataType;
```

Comment:
Since this typedef is only used once, I think it is best not to have a typedef at all, and just inline it into the next line.

Reply:
Done.